src/OrderTaking/PlaceOrder.Implementation.fs

```diff
@@ -1,4 +1,4 @@
-﻿module internal OrderTaking.PlaceOrder.Implementation
+﻿module OrderTaking.PlaceOrder.ImplementationTypes
```

Comment:
Changed his module to be public, containing only the types that are exposed in the PlaceOrder Api module (hence the name change). A submodule was created below that keeps the implementation section internal.